### ISSUE TITLE
[Feat] Support for different buttons when modifier keys are held

### ISSUE THREAD

**Issue by marquiswang** [olivernybroe/intellij-touch#17]
It would be really nice to switch touchbar button modes when various modifier keys (or key combinations) are held.

**Comment by olivernybroe** [comment]
Agreed, For doing this we have to think of a clever way of showing this in the preferences also.

**Comment by olivernybroe** [comment]
IntelliJ just releases 2018.2 and this version adds support for the touch bar, their edition has modifiers and can do basically everything we can do, the only thing they are missing is a refresh button.

**Comment by olivernybroe** [comment]
Re-opening the plugin as Android Studio does not have Touch bar and seems like users want this plugin to continue.
